lib/ecto_adapters_dynamodb/application.ex

```diff
@@ -12,7 +12,6 @@ defmodule Ecto.Adapters.DynamoDB.Application do
     children = [
       # Starts a worker by calling: Ecto.Adapters.DynamoDB.Worker.start_link(arg1, arg2, arg3)
       # worker(Ecto.Adapters.DynamoDB.Worker, [arg1, arg2, arg3]),
-      worker(Ecto.Adapters.DynamoDB.Cache, []),
```

Comment:
@bernardd why is this no longer necessary?

Reply:
The cache is now launched in a per-repo manner at `lib/ecto_adapters_dynamodb.ex:45`. Otherwise you could get different repos hitting the same cached table name in a global cache.